pkg/libratoaggregator/aggregator.go

```diff
@@ -0,0 +1,100 @@
+package libratoaggregator
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"sync"
+	"time"
+)
+
+type Gauge struct {
+	Name        string  `json:"name"`
+	MeasureTime int64   `json:"measure_time"`
+	Source      string  `json:"source"`
+	Count       int64   `json:"count"`
+	Sum         float64 `json:"sum"`
+	Min         float64 `json:"min",omitempty`
+	Max         float64 `json:"max",omitempty`
+	SumSquares  float64 `json:"max",omitempty`
+}
+
+type Aggregator struct {
+	User  string
+	Token string
+	C     chan map[string]float64
+}
+
+func New(user, token string) *Aggregator {
+	return &Aggregator{
+		User:  user,
+		Token: token,
+		C:     make(chan map[string]float64),
+	}
+}
+
+func (a Aggregator) Start() {
+	for {
+		select {
+		case m := <-a.C:
+			log.Print(m)
+		}
+	}
+}
+
+type Payload struct {
+	Gauges []Gauge `json:"gauges"`
+}
+
+type Client struct {
```

Comment:
most of this client stuff will get thrown away since we're simply building an aggregator